packages/ipfs/package.json

```diff
@@ -93,10 +93,10 @@
     "hapi-pino": "^6.1.0",
     "hashlru": "^2.3.0",
     "interface-datastore": "^1.0.2",
-    "ipfs-bitswap": "^1.0.0",
+    "ipfs-bitswap": "ipfs/js-ipfs-bitswap#chore/use-libp2p-0.28",
     "ipfs-block-service": "^0.17.1",
     "ipfs-core-utils": "^0.2.3",
-    "ipfs-http-client": "^44.1.0",
+    "ipfs-http-client": "44.0.3",
```

Comment:
This needs to be changed! When I rebased this PR with the `js-ipfs` repo changes from the last week, I started getting this error:

```
Error: Cannot find module 'ipfs-http-client/src/dht/find-peer'
Require stack:
- /Users/vsantos/work/pl/gh/ipfs/js-ipfs/node_modules/libp2p-delegated-peer-routing/src/index.js
- /Users/vsantos/work/pl/gh/ipfs/js-ipfs/packages/ipfs/src/cli/daemon.js
```

From my analysis  so far, using the `"ipfs-http-client": "^44.1.0"` will install `ipfs-http-client` in `js-ipfs/packages/ipfs/node_modules/ipfs-http-client` as a link to its folder, which should be correct! However, the require in [libp2p/js-libp2p-delegated-peer-routing/master/src/index.js#L4](https://github.com/libp2p/js-libp2p-delegated-peer-routing/blob/master/src/index.js#L4) does not work with the above error and the tests will not start to run. If I put this fixed version, the dependency will be installed in `js-ipfs/node_modules/ipfs-http-client` and the above error will not happen. I am not familiar with lerna dependency mgr and I could not understand yet the issue. @achingbrain @hugomrdias do you have any idea on the issue?

Reply:
For posterity, the problem here was that the delegate-* modules depend on a module that resolves to one in the `/packages` folder, which other modules in that folder also depend on.  So lerna creates symlinks for those modules but then fails to hoist the symlink to the root `node_modules` folder.

This is a bit of an edge case as our dependency tree is a little wonky in this case but it also seems like a bug in lerna.  The short-term solution is to add the delegate-* modules to the `nohoist` list in `lerna.json`.